stackrox/base/rbac.authorization.k8s.io/clusterroles/clusterrole.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
clusterRoles are globally accesible so we'll want to put these here in a directory for stackrox here: [https://github.com/operate-first/apps/tree/master/cluster-scope/base/rbac.authorization.k8s.io/clusterroles](https://github.com/operate-first/apps/tree/master/cluster-scope/base/rbac.authorization.k8s.io/clusterroles)

Reply:
will come back to this in 2 hours

Reply:
will come back to this after I separate the role bindings

Reply:
working on

Reply:
completed